cmd/csilvm/csilvm.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"google.golang.org/grpc"
 
-	"github.com/container-storage-interface/spec/lib/go/csi"
+	csi "github.com/container-storage-interface/spec/lib/go/csi/v0"
```

Comment:
the package is already named `csi`; there's no need for aliasing it to the same here (and elsewhere in this PR)

Reply:
This is in my top 5 least favourite features of go. I always alias the package explicitly so it is obvious when scrolling from far down in a file to the imports section what the import is. Otherwise you need to click through the potentially long list of imports' sources to find one where the package name matches the imported name. I really don't know why go allows this feature at all.

Reply:
With "this feature" i mean having the package name be different from the directory name in which it is defined. I mean, the language supports import aliases...

Reply:
If you don't mind I'd like to keep it this way as it is more explicit. The same goes for the other places I do this.

Reply:
ok, as long as its consistent everywhere in this project (which I think you've done)